packages/votingV2/abis/VotingV2.json

```diff
@@ -0,0 +1,1961 @@
+[
+  {
```

Comment:
I have had to remove all the elements with multi dimensional arrays (i.e. `uint256[2][]`) only present in the spam signalling mechanism. They are not supported by graph-node: [see this open issue](https://github.com/graphprotocol/graph-cli/issues/342)

This means that we wont be able to process `SignaledRequestsAsSpamForDeletion` events.